### ISSUE TITLE
a bug in openid@2.0.4

### ISSUE THREAD

**Issue by QingyunLiu** [havard/node-openid#159]
Hi, when I use openid@2.0.4, I got a exception:
```
/Users/gzliuqingyun/Projects/g18crashdump/crashdump/node_modules/openid/openid.js:98
  return _buffer(str, 'binary').toString('base64');
         ^

TypeError: binary is not a function
    at Function.from (native)
    at from (native)
    at _base64encode (/Users/gzliuqingyun/Projects/g18crashdump/crashdump/node_modules/openid/openid.js:98:10)
    at _bigIntToBase64 (/Users/gzliuqingyun/Projects/g18crashdump/crashdump/node_modules/openid/openid.js:107:10)
    at Object.openid.associate (/Users/gzliuqingyun/Projects/g18crashdump/crashdump/node_modules/openid/openid.js:609:35)
    at chooseProvider (/Users/gzliuqingyun/Projects/g18crashdump/crashdump/node_modules/openid/openid.js:817:16)
    at /Users/gzliuqingyun/Projects/g18crashdump/crashdump/node_modules/openid/openid.js:831:7
    at /Users/gzliuqingyun/Projects/g18crashdump/crashdump/node_modules/openid/openid.js:581:7
    at /Users/gzliuqingyun/Projects/g18crashdump/crashdump/node_modules/openid/openid.js:447:16
    at Request._callback (/Users/gzliuqingyun/Projects/g18crashdump/crashdump/node_modules/openid/openid.js:210:7)

```
After changed version to openid@2.0.1, it works fine.

**Comment by havard** [comment]
Which version of Node is this?

**Comment by QingyunLiu** [comment]
v5.5.0

**Comment by QingyunLiu** [comment]
OpenID provider is https://login.netease.com/openid/

**Comment by havard** [comment]
Fixed in 2.0.5.
